migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc

```diff
@@ -14,3 +14,5 @@ include::modules/migration-setting-up-target-cluster-to-accept-source-dns-domain
 .Additional resources
 
 * See xref:../security/certificates/replacing-default-ingress-certificate.adoc#replacing-default-ingress[Replacing the default ingress certificate] for more information.
+
+include::modules/migration-network-traffic-redirection-strategies.adoc[leveloffset=+2]
```

Comment:
@apinnick 
I've also changed the file name. Is it OK?

Reply:
@apinnick
Also, the Planning assembly, which is supposed to describe "considerations and strategies for planning your migration", now includes 'DNS considerations' and 'Network traffic redirecting strategies', both of which refer to activities after migration. Maybe the assembly should be renamed?

Reply:
We can't rename the assembly without having to set up redirects. Vikram would not be happy. We could change the title.

However, I think it should still be called Planning considerations even if the activities are done after migration, because the user needs to plan for them.

Reply:
New module name looks fine.

Reply:
apinnick
How about we change the assembly title to 'Networking considerations' and place it in MTC after 'Advanced migration options'?

Reply:
@apinnick
How about we change the assembly title to 'Networking considerations' and place it in MTC after 'Advanced migration options'?

Reply:
I think we need to discuss this over a video call, not in the PR.

Reply:
I think I would change the assembly title to "Network considerations" and leave it where it is, since its contents are now related to networking. You'll have to update the topic map file.

I would also duplicate the assembly for MTC. It is no longer a migration planning assembly.